test/features/labelDataDisplay.test.ts

```diff
@@ -0,0 +1,81 @@
+import { Tester } from '@mighty-justice/tester';
+import { COMPONENT_GENERATORS, fakeField, fakeTextShort } from '../factories';
+
+const normalFieldSet = { field: fakeField(), label: fakeTextShort(), type: 'string' },
```

Comment:
Looks like you could just use the factory:

```suggestion
const fieldSets = [[stringFactory.build()]],
```

And if you add the [[]] here and rename it `fieldSets` you can shorthand it below, like I showed with layout.